changes/21475-vpp-self-service-panic

```diff
@@ -0,0 +1 @@
+- Fixes a panic that occurs when an end user attempts to install a VPP app from self-service.
```

Comment:
If this is an unreleased bug I don't think we want a changes file

Reply:
Ah ok, removed!